src/permission/DefaultPermissions.php

```diff
@@ -106,6 +106,7 @@ public static function registerCorePermissions() : void{
 		self::registerPermission(new Permission(Names::COMMAND_UNBAN_IP, l10n::pocketmine_permission_command_unban_ip()), [$operatorRoot]);
 		self::registerPermission(new Permission(Names::COMMAND_UNBAN_PLAYER, l10n::pocketmine_permission_command_unban_player()), [$operatorRoot]);
 		self::registerPermission(new Permission(Names::COMMAND_VERSION, l10n::pocketmine_permission_command_version()), [$everyoneRoot]);
+		self::registerPermission(new Permission(Names::COMMAND_WEATHER, "Changing the weather in the world"));
```

Comment:
??

Reply:
a direct translation should be created `l10n::pocketmine_permission_command_weather()`

Reply:
I forgot to change the permission to the correct word